src/webgpu/api/validation/image_copy/layout_related.spec.ts

```diff
@@ -162,7 +162,11 @@ g.test('required_bytes_in_copy')
     const layout = { offset, bytesPerRow, rowsPerImage };
     const minDataSize = dataBytesForCopyOrFail({ layout, format, copySize, method });
 
-    const texture = t.createAlignedTexture(format, copySize);
+    const texture = t.createAlignedTexture(format, {
+      width: copyWidthInBlocks,
+      height: copyHeightInBlocks,
+      depthOrArrayLayers: copyDepth,
+    });
```

Comment:
I actually made createAlignedTexture take raw pixel size instead since taking sizes in block actually makes less sense. I also went and updated the other tests that were using it so that they make more sense now.